tests/errors/SettingsError/filters/f01.exec

```diff
@@ -0,0 +1,2 @@
+# format: exec
+exec:--filter;asdf=2016-02-01;
```

Comment:
Oh, I finally understood. I was counting the `-` shown by the diff as part of the argument :man_facepalming: 

Looks correct now.